css/README.md

```diff
@@ -239,19 +239,126 @@ These rules were adapted from [CSS Guidelines]. This is an example of how declar
+
+
+1. #### Included Code
+
+	1. Local `$variables`
+		* Local variables should really only be declared for mixins or functions. Most variables should be kept in the main variables partial, since variables in Sass are global.
+		* However, if you have a set of variables that will only be used in a certain partial, declare them at the top of that file and prefix their names to avoid confusion in the global namespace. e.g., `$m_module_name_height`.
+
+	1. `@extend` statements
+		* Avoid extends in favor of mixins. Extends offer no benefits over mixins, and can cause [specificity problems](http://csswizardry.com/2016/02/mixins-better-for-performance/).
+
+	1. `@include` statements
+		* Knowing right off the bat that this class inherits another whole set of rules from elsewhere is good. Another benefit is that overriding styles for that inherited set of rules becomes much easier.
+
+2. #### Alphabetized Regular Styles
+	1. Adding regular styles after the `@extend` and `@include` statements allows us to override those properties, if needed. Be sure to leave a comment if a declaration's only purpose is to override included styles. 
+    
+	1. Alphabetizing is helpful for a large team because it standardizes location of properties (See "Property Sorting" below).
+
+3. #### Modifiers
+	1. Pseudo-classes (`:hover`), attribute selectors (`&[type="input"]`), and state classes (`&.is-active`)
+		* These directly modify the parent selector so we declare them before other nested selectors.
+        * Most BEM modifier classes don't need to be nested. However, some generic state class names like `is-active` that would otherwise be too non-specific to live in the global namespace, should be nested here.
+
+	1. Nested media queries
+		* These come after regular styles, pseudo-classes, etc., so they can override them.
+        
+    1. Parent modifier classes (`.parent__modifier &`)
+        * Modifiers on parents can affect children. They should be nested like media queries.
+
+4. #### Child Selectors
+
+	1. Pseudo-elements (`::before`)
+		* Pseudo-elements are actually generated children, and so should be treated like any other nested selector.
+
+	2. Nested selectors
+		* As a rule, **if a selector will work without it being nested then do not nest it.**
+		* There shouldn’t be many nested selectors. Our naming convention means that we don’t need to nest selectors for namespacing. Before you nest a selector, consider if it would be better as a child class or modifier class.
+		* Any nested selectors could contain their own nested pseudo-classes, pseudo-elements and media queries, following the rules above.
+
+```sass
+.foo {
+    @include font_ui(2.0); // 1.iii
+    line-height: 1;        // override font_ui line-height // 2.i
+    width: 100%;           // 2.ii
```

Comment:
I noticed above there are numerical comments styled like this: 

`/* 1 */` 

and here comments are styled as:

`//1.iii 

Does that matter?

Also, I was confused by double, double slashes in this comment: 

`// override font_ui line-height // 2.i`

Is that intentional?

Reply:
Yeah, the `1.iii` references the way the ordered list numbrers are rendered in the document. When you view the rendered markdown, this makes it easier to tell which item it's referring to.

and no, the double comment was a mistake, I'll fix that.